tests/Arrays/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -10,20 +10,21 @@ contract Arrays {
 pragma solidity ^0.5.2;
 
 contract Arrays {
-    bytes32[] public permissions = [
-        bytes32("addPartner"),
-        bytes32("removePartner"),
-        bytes32("addGroup"),
-        bytes32("removeGroup"),
-        bytes32("addUserToGroup"),
-        bytes32("removeUserFromGroup"),
-        bytes32("addSolution"),
-        bytes32("removeSolution"),
-        bytes32("addLibrary"),
-        bytes32("removeLibrary"),
-        bytes32("addPermissionToGroup"),
-        bytes32("removePermissionFromGroup")
-    ];
+    bytes32[] public permissions =
+        [
```

Comment:
Are we sure we want this? I kind of prefer the previous version.

Reply:
I agree w/ that